core/src/main/java/bio/terra/pearl/core/service/participant/search/EnrolleeSearchService.java

```diff
@@ -62,4 +73,24 @@ protected EnrolleeSearchFacet getTaskFacet(StudyEnvironment studyEnvironment) {
         tasksFacet.addOptions(status);
         return tasksFacet;
     }
+
+    public Map<String, SearchValue.SearchValueType> getExpressionSearchFacetsForStudyEnv(UUID studyEnvId) {
+
+        Map<String, SearchValue.SearchValueType> fields = new HashMap<>();
+        // profile fields
+        ProfileTerm.FIELDS.forEach((term, type) -> fields.put("profile." + term, type));
+        // age
+        fields.put("age", SearchValue.SearchValueType.INTEGER);
```

Comment:
While testing locally, I'm getting an NPE when trying to save a survey with an expression that uses the `age` facet:

```
java.lang.NullPointerException: temporal
	at java.base/java.util.Objects.requireNonNull(Objects.java:235)
	at java.base/java.time.LocalDate.from(LocalDate.java:394)
	at java.base/java.time.LocalDate.until(LocalDate.java:1702)
	at java.base/java.time.Period.between(Period.java:389)
	at bio.terra.pearl.core.service.search.terms.AgeTerm.extract(AgeTerm.java:28)
	at bio.terra.pearl.core.service.search.expressions.EnrolleeTermComparisonFacet.evaluate(EnrolleeTermComparisonFacet.java:40)
	...
```